ui/lib/fetchNewsData.ts

```diff
@@ -1,9 +1,14 @@
+import fs from "node:fs/promises";
```

Comment:
Can you move this code to `ui/app/api/news/[id]/router.ts` for consistency reasons?